DuckDuckGoTests/FaviconsTests.swift

```diff
@@ -98,11 +98,34 @@ class FaviconsTests: XCTestCase {
     
     func testWhenGeneratingKingfisherResourceThenCorrectKeyAndURLAreGenerated() {
         
-        let resource = Favicons.shared.defaultResource(forDomain: "example.com")
-        XCTAssertEqual(resource?.cacheKey, "\(Favicons.Constants.salt)example.com".sha256())
+        let resource = Favicons.shared.defaultResource(forDomain: Constants.exampleDomain)
+        XCTAssertEqual(resource?.cacheKey, "\(Favicons.Constants.salt)\(Constants.exampleDomain)".sha256())
         XCTAssertEqual(resource?.downloadURL, URL(string: "https://example.com/apple-touch-icon.png"))
         
     }
+
+    func testWhenDomainIsBookmarkThenIsFaviconCached() {
+        let expectation = self.expectation(description: "isFaviconCachedForBookmarks")
+
+        let fileName = "MockFiles/DDG-icon_256x256.png"
+
+        guard let imageData = try? FileLoader().load(fileName: fileName, fromBundle: Bundle(for: type(of: self))),
+              let image = UIImage(data: imageData),
```

Comment:
Ah, I didn't know about Logo - thanks for letting me know.  I'm happy to do this if you prefer; however if the `Logo` filename is every changed then the test will break.  The logo I added to mockdata is for the test only.  If we have the unit test use production `Logo`, it would be similar to creating a UnitTest that tests production data (although I can appreciate that in this case the `Logo` name is unlikely to change as often compared to something like prod JSON data).  So it's a small code smell but not too smelly so I don't have too strong an opinion

Reply:
No problem, I've made that change :)